examples/github.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const fastify = require('fastify')({ logger: { level: 'trace' } })
+const sget = require('simple-get')
+
+// const oauthPlugin = require('fastify-oauth2')
+const oauthPlugin = require('..')
+
+fastify.register(oauthPlugin, {
+  name: 'githubOAuth2',
+  scope: [],
+  credentials: {
+    client: {
+      id: '<CLIENT_ID>',
+      secret: '<CLIENT_SECRET>'
+    },
+    auth: oauthPlugin.GITHUB_CONFIGURATION
+  },
+  startRedirectPath: '/login/github',
+  callbackUri: 'http://localhost:3000/login/github/callback'
+})
+
+let refreshToken = ''
+
+fastify.get('/login/github/callback', function (request, reply) {
+  this.githubOAuth2.getAccessTokenFromAuthorizationCodeFlow(
+    request,
+    (err, result) => {
+      if (err) {
+        reply.send(err)
+        return
```

Comment:
Don't think the returns are needed after the reply.send()s.

Reply:
it is used to do not execute the `sget.concat`